src/main/resources/application.properties

```diff
@@ -157,3 +157,6 @@ duos.basePath=https://consent.dsde-dev.broadinstitute.org
 tps.basePath=https://tps.dsde-dev.broadinstitute.org
 sentry.dsn=
 sentry.environment=undefined
+notification.project-id=broad-dsde-dev
+notification.topic-id=workbench-notifications-dev
+# prod: broad-dsde-prod, workbench-notifications-prod
```

Comment:
Presumably terraform will override these? Just to confirm my understanding?

Reply:
Yes, I need to PR into `terra-helmfile` (in https://github.com/broadinstitute/terra-helmfile/blob/master/values/app/datarepo/live/prod.yaml#L96) to add this to prod. I'll fix that before merging up.

Reply:
can you explain this more just for my understanding? These are temporary changes that will be removed once the PR to terra-helmfile is merged? Or once we test this on dev, we will want to update these values to be prod values?

Reply:
I'll remove this comment now. These settings are valid for all environments except prod, so there's no need to change it in a later PR. When I add prod messages, I'll add the prod settings to helmfile, which will override these settings in the prod environment